index.html

```diff
@@ -1,19 +1,115 @@
 
 <html lang="en">
+
 <head>
   <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
 
-  <title>My Resume</title>
+  <title>John Wick - Resume</title>
 
+  <link href="https://fonts.googleapis.com/css?family=Gugi|Open+Sans|Poppins" rel="stylesheet">
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css" integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf" crossorigin="anonymous">
   <link rel="stylesheet" href="css/index.css">
-
-  <!--[if lt IE 9]>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
-  <![endif]-->
 </head>
 
 <body>
-  <h1>My Resume</h1>
+  <div class="main-container">
+    <header>
+      <nav>
+        <ul>
+          <li><a href="#intro">Intro</a></li>
+          <li><a href="#skills">Skills</a></li>
+          <li><a href="#work">Work</a></li>
+          <li><a href="#work-and-contact">Contact</a></li>
+        </ul>
+      </nav>
+    </header>
+
+    <section class="bio">
+      <img src="images/jw.jpg" alt="john-wick" />
+
+      <div class="job-title">
+        <h1>John Wick</h1>
+
+        <h2>Full Stack Javascript Developer</h2>
+
+        <article id="intro">
+          <p>A world class developer with years of industry experience crafting innovative
+            solutions. I have collaborated and interacted with industry expert and ganered the 
+            requisite experience required to slot into any technical position.
+          </p>
+        </article>
+      </div>
+    </section>
+
+    <hr />
+
+    <section id="skills">
+      <h2>Skills</h2>
+      <p>The best form of learning is to learn by doing. Through years of world class
+        performance I have acquired the following skills and competencies:
+      </p>
+
+      <h3>Languages</h3>
+      <ul>
+        <li>Javascript</li>
+        <li>Python</li>
+        <li>C</li>
+      </ul>
+
+      <h3>Frameworks and Libraries</h3>
+      <ul>
+        <li>Express</li>
+        <li>React and Redux</li>
+        <li>JQuery</li>
+        <li>GraphQL</li>
+        <li>Django</li>
+      </ul>
+    </section>
+
+    <hr />
+
+    <section id="work">
+      <h2>Work History</h2>
+      
+      <table id="company">
+        <tr class="head"><th>Company Name</th><th>Position</th><th>Years</th></tr>
+        <tr><td>Google</td><td>Software Engineer</td><td>2</td></tr>
+        <tr><td>Microsoft</td><td>Backend Developer</td><td>3</td></tr>
+        <tr><td>Affirm</td><td>Software Developer</td><td>5</td></tr>
+      </table>
+
+    </section>
+
+    <hr />
+
+    <section id="work-and-contact">
+      <div class="contact">
+        <h2>Contact</h2>
+        <p>Telephone: +23400000000</p>
+        <p>Email: j_w@john.com</p>
+        <p>LinkedIn: linkedin.com/in/j_w</p>
+        <div><i class="fab fa-twitter-square"></i><i class="fab fa-facebook-square"></i><i class="fab fa-github-square"></i></div>
```

Comment:
Your HTML looks awesome. Really professional. 
The only little thing I would probably do is break this line. 
It feels too long, especially if you plan to attach an `<a>` tag to it, later on.